plugins/codemirror/plugin.js

```diff
@@ -129,17 +126,22 @@
 				CKEDITOR.plugins.codemirror.commands.source
 			);
 
+			var codeMirrorDialogConfigs = {
+				state: CKEDITOR.TRISTATE_OFF,
+			};
+
+			if (!editor.balloonToolbars) {
+				codeMirrorDialogConfigs['modes'] = {
```

Comment:
Non-blocking, minor SF: If object property is a constant, we should use dot notation, `codeMirrorDialogConfigs.modes`. That way we don't declare a new string and gain a tiny bit on performance.

Reply:
Makes sense. I'll change. I remember seeing this [pattern in linktoolbar](https://github.com/liferay/liferay-portal/blob/b6e202c32fb58fca5a54f9b4b5adb21351f67a42/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/linktoolbar/plugin.js#L101)